e2e/cypress/support/api_commands.js

```diff
@@ -234,6 +234,22 @@ Cypress.Commands.add('apiUnpinPosts', (postId) => {
     });
 });
 
+/**
+ * Get a post
+ * This API assume that the user is logged in and has cookie to access
+ * @param {String} post_id - The post id of the post
+ */
+Cypress.Commands.add('apiGetPost', (postId) => {
```

Comment:
💯 Nice!
We're currently organizing API commands and will eventually move everything to `/support/api` folder. With that, please move this into a new file called `/support/api/post.js` and add type definition into new `/support/api/post.d.ts`. See https://github.com/mattermost/mattermost-webapp/blob/master/e2e/cypress/support/api/team.d.ts for reference.
 (We have it on item 2 in our [docs](https://developers.mattermost.com/contribute/webapp/end-to-end-tests/#writing-specs) 😄 )

Reply:
I've removed `apiGetPost` for now since we've decided to verify the user on the UI. I'll keep the location of the API command in mind. Now I know where to look for them if I need to use them. 😄